src/api/json/catalog.json

```diff
@@ -2558,6 +2558,15 @@
       "description": "Traefik v2 Dynamic Configuration File Provider",
       "url": "https://json.schemastore.org/traefik-v2-file-provider.json"
     },
+    {
+      "name": "Transparency Information Language and Toolkit (TILT)",
+      "description": "A transparency information language and toolkit explicitly designed to represent and process transparency information in line with the GDPR.",
+      "url": "https://json.schemastore.org/tilt.json",
+      "fileMatch": [
+        "*.tilt.json",
+        "*tilt*.json"
```

Comment:
"*tilt*.json" Is this necessary? This can intercept JSON files that are not tilt schema.

Reply:
Thanks! I'll fix this soon!